contracts/src/CeramicAnchorServiceV2.sol

```diff
@@ -17,15 +17,24 @@ contract CeramicAnchorServiceV2 is Ownable {
 
     //upon successful anchor
     event DidAnchor(address indexed _service, bytes _root);
+    // event DidAnchor(address indexed _service, bytes32 _root);
 
     // Only an address in the allow list is allowed to anchor
+    // TODO: upon contract creation, allowList is blank, so nobody is allowed
```

Comment:
If you have a TODO like this, please, accompany it with a corresponding Linear issue, which tracks the fix.

Like this: `TODO(NET-NNNN): upon ....`